packages/schedule/src/index.ts

```diff
@@ -5,5 +5,5 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-export { default as useSchedule } from './useSchedule';
-export { default as ScheduleContainer } from './ScheduleContainer';
+export { useSchedule, IUseScheduleReturnValue } from './useSchedule';
```

Comment:
This should also export its attribute props interface `IUseScheduleProps`.

Reply:
`IUseScheduleProps` wasn't created because `useSchedule` arguments types were inferred via their default arguments. But I can see why it's better to create an interface and export it. I'll go ahead and do that. Thanks for the suggestion!

Reply:
Updated, thanks for the suggestion!